hi-ml-cpath/src/health_cpath/utils/report_utils.py

```diff
@@ -237,3 +258,21 @@ def collect_class_info(metrics_df: pd.DataFrame) -> Tuple[int, List[str]]:
         class_names = [name.lstrip() for name in class_names[1:-1].replace("'", "").split(',')]
     class_names = validate_class_names(class_names=class_names, n_classes=num_classes)
     return (num_classes, list(class_names))
+
+
+def collect_epoch_info(metrics_df: pd.DataFrame) -> Dict[int, int]:
```

Comment:
I'd suggest renaming this function `get_max_epochs()` or similar to make its purpose crystal-clear.

Reply:
good idea, renamed now